pkg/sippyserver/server.go

```diff
@@ -66,9 +66,10 @@ type TestGridDashboardCoordinates struct {
 }
 
 type StandardReport struct {
-	CurrentPeriodReport sippyprocessingv1.TestReport
-	CurrentTwoDayReport sippyprocessingv1.TestReport
-	PreviousWeekReport  sippyprocessingv1.TestReport
+	CurrentPeriodReports []sippyprocessingv1.TestReport
+	CurrentPeriodReport  sippyprocessingv1.TestReport
```

Comment:
this ought not be needed, right?

You could simplify the diff by creating getters, right?

Reply:
yep will just require more refactoring so wanted to make sure the approach was sound before I went too deep